assets/syslog-drain-listener/syslog_drain.go

```diff
@@ -24,6 +57,25 @@ func main() {
 	}
 }
 
+type Credentials struct {
+	CA   string `json:"ca"`
+	Key  string `json:"key"`
+	Cert string `json:"Cert"`
+}
+
+func getCreds() Credentials {
+	mtlsString := os.Getenv("MTLS")
+	if mtlsString == "" {
```

Comment:
Is this check required? Won't unmarshalling with an empty string just create an empty credentials object?

Reply:
Yes, this check appears to be required.